packages/bpk-docs/src/pages/ModalsPage/ModalsPage.js

```diff
@@ -131,6 +133,46 @@ const components = [
       </ModalContainer>,
     ],
   },
+  {
+    id: 'no-full-screen-on-mobile',
```

Comment:
We intentionally want to hide this example from the Modals docs page and only show it in the Dialog docs page (when it exists)

Reply:
Done.